docs/LEARNING.md

```diff
@@ -2,6 +2,7 @@
 
 * [The Rust Programming Language](http://doc.rust-lang.org/stable/book/) is a great resource for getting started with Rust as well as diving deeper into specific features of Rust.
 * [Rust by Example](http://rustbyexample.com) shows you examples of the most common things you will be writing in Rust.
+* [Into_rust()](http://intorust.com/) "Screencasts for learning Rust."
```

Comment:
This seems appropriate, then. It looks like the below were also direct quotes: https://www.rust-lang.org/en-US/community.html and https://github.com/carols10cents/rustlings also contain the exact quotes listed for the below entries. Since the used text is also a quote, we are now being consistent.